src/_pytest/config/__init__.py

```diff
@@ -757,6 +782,11 @@ def _mark_plugins_for_rewrite(self, hook):
 
         self.pluginmanager.rewrite_hook = hook
 
+        load_entrypoint_plugins = self.getini("load_entrypoint_plugins")
+        if load_entrypoint_plugins is not notset and not len(load_entrypoint_plugins):
+            assert isinstance(load_entrypoint_plugins, list)
```

Comment:
Can we have a more friendly message than an `assert`? For example if a user writes `load_entrypoint_plugins=xdist`, the option will be read as a string and the assert will blow up without any useful information.

Reply:
Actually see my tip above about using `type="args"` in the option declaration. 👍